Actions/Deploy/Deploy.ps1

```diff
@@ -115,7 +145,7 @@ else {
         $sandboxEnvironment = ($response.environmentType -eq 1)
         $scope = $deploymentSettings.Scope
         if ($null -eq $scope) {
-            if ($settings.Type -eq 'AppSource App' -or ($sandboxEnvironment -and !($bcAuthContext.ClientSecret -or $bcAuthContext.ClientAssertion))) {
+            if ($settings.type -eq 'AppSource App' -or ($sandboxEnvironment -and !($bcAuthContext.ClientSecret -or $bcAuthContext.ClientAssertion))) {
```

Comment:
Is it not lowercase? 
https://github.com/microsoft/AL-Go/blob/84a0be1625e0055e72cc796299d8ec5710416f60/Actions/.Modules/ReadSettings.psm1#L105

Reply:
Yes. It is lowercase. The change is from $settings.**T**ype to $settings.**t**ype.

Reply:
Right, you are 😆 Lesson to self; Don't review PRs in the late afternoon